faqs/galaxy/histories_shared.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Finding and working with "Histories shared with me"
+description: How to find and work on histories shared with you
+area: datasets
+layout: faq
+box_type: tip
+contributors: [jennaj, garimavs]
+---
+
+**To find histories shared with me:**
+- Log into your account.
+- Select **User**, in the drop-down menu, select **Histories shared with me**.
+
+**To work with shared histories:**
+- Import the History into your account via copying it to work with it.
+- Unshare Histories that you no longer want shared with you or that you have already made a copy of.
```

Comment:
are these also steps?

Reply:
To work with shared histories, you need to copy them and then work on them, unsharing history part is more related to the account quota management thing.